orchagent/port/porthlpr.cpp

```diff
@@ -1029,6 +1032,11 @@ bool PortHelper::parsePortConfig(PortConfig &port) const
         }
         else
         {
+            if (unknown.find(field) != unknown.end())
+                continue; // do not pollute logs with the same unknown field
```

Comment:
@kcudnik this won't be working since we have two modes: 1. Aggregation mode before the port is created 2. Parameter(s) processing mode after port is created. Thus, you will have to maintain a permanent storage to filter the key-value pairs to avoid duplicated logs (relevant for the port create stage). It should be disposed once port is created/removed

Reply:
so let's move set to class member ?